model/src/pyrenew/deterministic/deterministic.py

```diff
@@ -41,18 +41,14 @@ def validate(vars: tuple) -> None:
 
     def sample(
         self,
-        random_variables: dict = None,
-        constants: dict = None,
+        **kwargs,
     ) -> tuple:
         """Retrieve the value of the deterministic Rv
 
         Parameters
         ----------
-
-        random_variables : dict
-            Ignored. Default None.
-        constants : dict
-            Ignored. Default None.
+        kwargs : dict
+            Ignored.
```

Comment:
Agree with @AFg6K7h4fhy2 that I think it would be best to say something like 
```Python
"""
**kwargs : dict, optional
    Additional keyword arguments passed through to internal `sample()` calls, if any
"""
```

Reply:
In some cases, the kwargs are explicitly ignored, i.e., not passed to anything. But we keep them for consistency. Should I still say these could be passed? That doesn't seem right.

Reply:
Clarification. As I understand it, the pattern we're planning to use is the following:

```Python
class Example(RandomVariable):
   
    def __init__(self):
           self.internal_rv_ = OtherRandomVariable()

    def sample(self, mandatory_named_arg: ArrayLike = None, **kwargs):
           self.do_something_with_named_arg(mandatory_named_arg)
           self.interval_rv_.sample(**kwargs) 
```

So any `RandomVariable.sample()` method only refers to named arguments within its own function body, but it also passes any additional `**kwargs` it receives onward to any calls to other `RandomVariable.sample() methods` that happen under the hood.